data/fixtures/settings.yml

```diff
@@ -3,7 +3,7 @@ QubitSetting:
     name: version
     editable: 0
     deleteable: 0
-    value: 193
```

Comment:
I don't know if a migration's actually needed for this as I think your call to `parent::earlyExecute();` will end up creating the new setting in the DB.

Reply:
initially, we started out with a global setting so we created a migration. if it's not needed, what changes do i need to make?

Reply:
I'd try using the setting in a fresh install of AtoM without running the migration and see if it works. If it works without the migration then all you need to do is remove the changes settings.yml from your dev change (and remove the actual migration file).

Reply:
Nicely done! Looks good!